docs/resources/mcs_db_instance.md

```diff
@@ -74,12 +74,16 @@ The following arguments are supported:
 * `disk_autoexpand` - Object that represents autoresize properties of the instance. It has following attributes:
     * `autoexpand` - Boolean field that indicates whether autoresize is enabled.
     * `max_disk_size` - Maximum disk size for autoresize.
+  
+* `wal_disk_autoexpand` - Object that represents autoresize properties of the instance. It has following attributes:
+    * `autoexpand` - Boolean field that indicates whether wal volume autoresize is enabled.
+    * `max_disk_size` - Maximum disk size for wal volume autoresize.
 
 * `wal_volume` - Object that represents wal volume of the instance. Changing this creates a new instance. It has following attributes:
     * `size` - (Required) Size of the instance wal volume.
     * `volume_type` - (Required) The type of the instance wal volume. Changing this creates a new instance.
-    * `autoexpand` - Boolean field that indicates whether wal volume autoresize is enabled.
-    * `max_disk_size` - Maximum disk size for wal volume autoresize.
+    * `autoexpand` - Boolean field that indicates whether wal volume autoresize is enabled. **Deprecated** Please, use wal_disk_autoexpand block instead.

```

Comment:
i'd prefer to use something like **deprecated since version ...** (v0.6.0 i guess) here and bellow

Reply:
but if it's more about backend, maybe it's better not to use version, okay